Dockerfile

```diff
@@ -153,9 +154,7 @@ RUN apk add --no-cache --virtual .rubymakedepends \
   "barby:${BARBY_VERSION}" \
   "rqrcode:${RQRCODE_VERSION}" \
   "chunky_png:${CHUNKY_PNG_VERSION}" \
-  && apk del -r --no-cache .rubymakedepends \
-  # Fixes an issue with 2 nokogiri versions breaking asciidoctor-epub3 on arm64
-  && if [ "$TARGETARCH" = "arm64" ]; then gem uninstall nokogiri -v '> 1.14'; fi
```

Comment:
Removing the workaround from https://github.com/asciidoctor/docker-asciidoctor/pull/407 as it looks like `asciidoctor-epub3`  tests passes with the latest native `nokogiri` Alpine 3.21 package (even on  `arm64`)